### ISSUE TITLE
Entity collections with zero children result in no action on import

### ISSUE THREAD

**Issue by twistedstream** [auth0/auth0-deploy-cli#74]
Let's say you have a `tenant.yaml` that contains entity collections with zero children. For example:

```yaml
clients: []
rules: []
connections: []
```

If the `AUTH0_ALLOW_DELETE` environment variable was set to `true` and you peformed an Import, the expected behavior would be that the Deploy CLI would remove _all_ existing entities from the tenant (eg. the YAML above would remove all existing Rules and Clients). However, the actual behavior is that those entities are ignored (skipped). This makes it impossible to craft a YAML file that wipes all entities from a tenant, which is useful in certain circumstances.

Ignoring an entity is also useful. I suggest for that we do so if a particular entity collection is simply not present in the YAML. For example, if you wanted your YAML to ignore rules, then the above YAML file change to this:

```yaml
clients: []
connections: []
```
### NOTE
Making the above suggested behavioral change to the CLI could be destructive to users who cuurently have YAML files that don't follow this paradigm. We may want to introduce a new environment variable to trigger it (eg. `AUTH0_ALLOW_DELETE_ALL_ENTITIES_IN_EMPTY_COLLECTION`).

I haven't tested it, but this may also be an issue with the `directory` format as well.

**Comment by esalter** [comment]
Ran into this as well during testing, would be great to see support for this.

**Comment by fyockm** [comment]
fix in #89 is now published as `auth0-deploy-cli@2.3.0`
